java/ql/lib/semmle/code/java/dataflow/FlowSources.qll

```diff
@@ -36,6 +36,14 @@ abstract class RemoteFlowSource extends DataFlow::Node {
   abstract string getSourceType();
 }
 
+/**
+ * A module for importing frameworks that define remote flow sources.
+ */
+private module RemoteFlowSources {
+  private import semmle.code.java.frameworks.android.Widget
+  private import semmle.code.java.security.TemplateInjection
```

Comment:
I don't think `TemplateInjection` defines new remote sources. It only has:

https://github.com/github/codeql/blob/ef9b503f69bb7e9f7f5d91a7dce2520d8b3f6380/java/ql/lib/semmle/code/java/security/TemplateInjection.qll#L65

Which is a remote flow source _use_, not a new definition.

```suggestion
```

Reply:
Oh - you are right! Good catch! Will fix.
I just searched for regexp `extends (.*)RemoteFlowSource` and did not consider this case (and didn't pay attention to the actual source code).